pages/common/tlmgr-remove.md

```diff
@@ -0,0 +1,25 @@
+# tlmgr remove
+
+> Uninstall TeX Live packages.
+> By default, removed packages will be backed up to `.../tlpkg/backups` under the TL installation directory.
```

Comment:
```suggestion
> By default, removed packages will be backed up to `../tlpkg/backups` under the TL installation directory.
```

 should it not be 2 dots, to go back a dir

Reply:
This is not meant to go back one dir, but know where you say it it looks a bit confusing.
```suggestion
> By default, removed packages will be backed up to `tlpkg/backups` under the TL installation directory.
```
I assume this makes more sense right?

Reply:
yes

Reply:
or
```suggestion
> By default, removed packages will be backed up to `./tlpkg/backups` under the TL installation directory.
```